tox.ini

```diff
@@ -53,20 +47,9 @@ platform =
     windows: win32
     macOS: darwin
     linux: linux
-extras =
-    dev
-    !nosubset: gis
+extras = dev
 install_command = python -m pip install --no-user {opts} {packages}
 download = True
-deps =
-; If you want to make tox run the tests with the same versions, create a
-; requirements.txt with the pinned versions and uncomment the following line:
-    coveralls
```

Comment:
I would re-add coveralls. It would be good to get this check working again.

Reply:
haha so polite. My bad!